Nodejs/Product/Nodejs/Project/DependencyNodeProperties.cs

```diff
@@ -143,14 +143,6 @@ public class DependencyNodeProperties : NodeProperties {
 
         internal bool IsGlobalInstall {
             get {
-                var node = DependencyNode as HierarchyNode;
-                while (null != node) {
-                    if (node is GlobalModulesNode) {
-                        return true;
-                    }
-
-                    node = node.Parent;
-                }
                 return false;
```

Comment:
Agreed. We should remove it unless we can think of a particular case in which we would want to light up that codepath at some point in the future (personally, I can't think of a good reason given the typical usage of global modules.) My only concern is that those changes might be a tad more difficult to properly code review and I don't want to block on that for RTM. What do you think about adding a TODO for now so we can kick off a dev build, and then just follow it up with another PR?